cmd/run_controller/main.go

```diff
@@ -56,27 +55,27 @@ func main() {
 
 	system.Namespace() // ensure that namespace is set in environment
 
-	log.Printf("Create Factory (resync period: %s, QPS: %d, burst: %d)", resyncPeriod.String(), qps, burst)
+	klog.Infof("Create Factory (resync period: %s, QPS: %d, burst: %d)", resyncPeriod.String(), qps, burst)
 	config.QPS = float32(qps)
 	config.Burst = burst
 	factory := k8s.NewClientFactory(config, resyncPeriod)
 
-	log.Printf("Provide metrics")
+	klog.Infof("Provide metrics")
 	metrics := metrics.NewMetrics()
 	metrics.StartServer()
 
-	log.Printf("Create Controller")
+	klog.Infof("Create Controller")
 	controller := runctl.NewController(factory, metrics)
 
-	log.Printf("Create Signal Handler")
+	klog.Infof("Create Signal Handler")
 	stopCh := signals.SetupSignalHandler()
 
-	log.Printf("Start Informer")
+	klog.Infof("Start Informer")
 	factory.StewardInformerFactory().Start(stopCh)
 	factory.TektonInformerFactory().Start(stopCh)
 
-	log.Printf("Run controller")
+	klog.Infof("Run controller")
 	if err = controller.Run(2, stopCh); err != nil {
-		log.Fatalf("Error running controller: %s", err.Error())
+		klog.Fatalf("Error running controller: %s", err.Error())
 	}
 }
```

Comment:
```suggestion
  klog.Flush()
}
```

Reply:
Use `defer` to run it whenever the main method exits!